views/validation.ejs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <title>Validation</title>
+</head>
+<body>
+    <h1>Form Validation</h1>
+        <% if(count > 0) {%>
+            <div class="form-group p-3 mb-2 bg-danger text-white">
+                <h2>Email Already Exist</h2>
+            </div>
+            <form action="/signup" method="GET">
+                <button type="submit" class="btn btn-primary btn-sm">Go to SignUp Page</button>
+            </form>
+            <% } %>
+            <% if(count === 0) {%>
+                <div class="form-group p-3 mb-2 bg-success text-white">
+                    <h2>Registration Successful</h2>
+                </div>
+                <form action="/login" method="GET">
+                    <button type="submit" class="btn btn-primary btn-sm">Go to LogIn Page</button>
+                </form>
+                <% } %>
+</body>
+</html>
```

Comment:
Alignment is bad.  Not just this file.  But across all file.  Consistency is missing.  Some time it is 4 spaces.. and some time is is 2 space...

Reply:
yeah I will correct the alignment